viite-UI/src/router.js

```diff
@@ -71,7 +71,7 @@
         eventbus.trigger('underConstructionRoads:toggleVisibility', false);
         $('.underconstruction-visible-wrapper').hide();
         $('#toggleEditMode').hide();
-        $('#emptyFormDiv,#projectListButton').hide();
+        $('#emptyFormDiv,#formProjectButton').hide();
```

Comment:
Can stay as: `$('#emptyFormDiv').hide();`

I think _historyLayer_ is not being used at this moment.

If so, I think we should ask Janne if we want to hide the main project button (on the right)